waku/protocol/v2/filter.nim

```diff
@@ -1,37 +0,0 @@
-import
```

Comment:
Strange that this doesnt show up as a rename of file

Reply:
Git!